llvm/include/llvm/LinkAllPasses.h

```diff
@@ -207,6 +208,7 @@ namespace {
       (void) llvm::createExpandMemCmpPass();
       (void) llvm::createExpandVectorPredicationPass();
       (void)llvm::createESIMDVerifierPass();
```

Comment:
I see you has not touched this line, but maybe it could be aligned with the line above and (new one) below? What about to add a space after `(void)`?

Reply:
In fact, I realized that clang-format prefers `(void)llvm::create();` without whitespace.